<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="DavaGLWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuResentScenes">
     <property name="title">
      <string>Resent Scenes</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionPNG"/>
     <addaction name="actionPVR"/>
     <addaction name="actionDXT"/>
    </widget>
    <addaction name="actionNewScene"/>
    <addaction name="separator"/>
    <addaction name="actionOpenScene"/>
    <addaction name="actionOpenProject"/>
    <addaction name="menuResentScenes"/>
    <addaction name="separator"/>
    <addaction name="actionSaveScene"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuCreateNode">
    <property name="title">
     <string>Create Node</string>
    </property>
    <addaction name="actionLandscape"/>
    <addaction name="actionLight"/>
    <addaction name="actionServiceNode"/>
    <addaction name="actionBox"/>
    <addaction name="actionSphere"/>
    <addaction name="actionCamera"/>
    <addaction name="actionImposter"/>
    <addaction name="actionParticleEmitter"/>
    <addaction name="actionUserNode"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionMaterialEditor"/>
    <addaction name="actionTextureConverter"/>
    <addaction name="separator"/>
    <addaction name="actionHeightMapEditor"/>
    <addaction name="actionTileMapEditor"/>
   </widget>
   <widget class="QMenu" name="menuViewPort">
    <property name="title">
     <string>View Port</string>
    </property>
    <addaction name="actionIPhone"/>
    <addaction name="actionRetina"/>
    <addaction name="actionIPad"/>
    <addaction name="actionDefault"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRestoreViews"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuCreateNode"/>
   <addaction name="menuTools"/>
   <addaction name="menuViewPort"/>
  </widget>
  <widget class="QDockWidget" name="dockSceneGraph">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Scene Graph</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="sceneGraphBody">
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="sceneGraphTree"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveRootNodes">
       <property name="text">
        <string>Remove Root Nodes</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLockAtObject">
       <property name="text">
        <string>Lock at Object</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveObject">
       <property name="text">
        <string>Remove Object</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDebugFlags">
       <property name="text">
        <string>Debug Flags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBakeMatrices">
       <property name="text">
        <string>Bake Matrices</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBuildQuadTree">
       <property name="text">
        <string>Build Quad Tree</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockDataGraph">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>42</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Data Graph</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dataGraphBody"/>
  </widget>
  <widget class="QDockWidget" name="dockLibrary">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>42</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Library</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="libraryBody"/>
  </widget>
  <widget class="QDockWidget" name="dockProperties">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>42</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="propertiesBody"/>
  </widget>
  <widget class="QDockWidget" name="dockEntities">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>42</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Entity Graph</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="entityBody"/>
  </widget>
  <action name="actionNewScene">
   <property name="text">
    <string>New Scene</string>
   </property>
  </action>
  <action name="actionOpenScene">
   <property name="text">
    <string>Open Scene</string>
   </property>
  </action>
  <action name="actionSaveScene">
   <property name="text">
    <string>Save Scene</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionPVR">
   <property name="text">
    <string>PVR</string>
   </property>
  </action>
  <action name="actionDXT">
   <property name="text">
    <string>DXT</string>
   </property>
  </action>
  <action name="actionLandscape">
   <property name="text">
    <string>Landscape</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionServiceNode">
   <property name="text">
    <string>Service Node</string>
   </property>
  </action>
  <action name="actionBox">
   <property name="text">
    <string>Box</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionImposter">
   <property name="text">
    <string>Imposter</string>
   </property>
  </action>
  <action name="actionUserNode">
   <property name="text">
    <string>User Node</string>
   </property>
  </action>
  <action name="actionMaterialEditor">
   <property name="text">
    <string>Material Editor</string>
   </property>
  </action>
  <action name="actionHeightMapEditor">
   <property name="text">
    <string>Height Map Editor</string>
   </property>
  </action>
  <action name="actionTileMapEditor">
   <property name="text">
    <string>Tile Map Editor</string>
   </property>
  </action>
  <action name="actionTextureConverter">
   <property name="text">
    <string>Texture Converter</string>
   </property>
  </action>
  <action name="actionIPhone">
   <property name="text">
    <string>iPhone</string>
   </property>
  </action>
  <action name="actionRetina">
   <property name="text">
    <string>Retina</string>
   </property>
  </action>
  <action name="actionIPad">
   <property name="text">
    <string>iPad</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionParticleEmitter">
   <property name="text">
    <string>Particle Emitter</string>
   </property>
  </action>
  <action name="actionRestoreViews">
   <property name="text">
    <string>Restore Views</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DavaGLWidget</class>
   <extends>QWidget</extends>
   <header>davaglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
